Migration.md

```diff
@@ -0,0 +1,285 @@
+# Migrate from Lob’s JavaScript to TypeScript SDK
+
+This guide illustrates differences between Lob’s current JavaScript library (lob-node) and our new TypeScript SDK (lob-typescript-sdk). The new TypeScript SDK supports all generally available endpoints and has the benefit of being built with TypeScript.
+We have also compared performance and found the new TypeScript SDK is equivalent or improved in most cases with some operations experiencing a consistent 2X improvement.
+In this guide we compare the following method pattern.
```

Comment:
Add the following above "In this guide ..."

Note: apps written in JavaScript or TypeScript can both utilize Lob’s TypeScript SDK